### ISSUE TITLE
How can i update the editor, if i want to load other text 

### ISSUE THREAD

**Issue by langley2k** [andrerpena/react-mde#109]
Hey,

in this sample code you are take the content inside the editor and duplicate the text.
 `
const newMdeState = await DraftUtil.buildNewMdeState(
      mdeState,
      this.generateMarkdownPreview,
      mdeState.markdown + " " + mdeState.markdown
    );
`

but in version 5.5.1 it doesn't work. i would go down to 5.1.1. but theres no tag anymore to older versions.
could you please help me, i´ve tried some hourse to get myself...

**Comment by andrerpena** [comment]
@langley2k , thanks for reporting. I'll look into it now

**Comment by rnemanja** [comment]
Any news on this?

**Comment by andrerpena** [comment]
Hey @rnemanja. Sorry for the delay. I've been quite busy with my personal life lately and could not give much attention to react-mde lately. I'm back. I was working on https://github.com/andrerpena/react-mde/issues/111 but apparently it does not have an easy solution. I'll focus on this issue now

**Comment by ythecombinator** [comment]
Hey @andrerpena, how do ya? 

I'm facing the very same issue here; any news on this?

**Comment by andrerpena** [comment]
@ythecombinator I'm working on #120 now... The value will be a simple text with only the markdown being passed as a parameter. This will hopefully solve this issue.

**Comment by devonnuri** [comment]
For temporary, use this code for edit text in MdeEditor..

It is just working, i think it could be shorten..

```javascript
const { mdeState } = this.state;
const newDraftState= DraftUtil.buildNewDraftState(
  mdeState.draftEditorState,
  {
    selection: {
      start: 0,
      end: 0
    },
    text: this.state.mdeState.markdown + ' ' + this.state.mdeState.markdown
  }
);
this.setState({
  mdeState: {
    markdown: mdeState.markdown,
    html: mdeState.html,
    draftEditorState: newDraftState
  }
});
```

**Comment by andrerpena** [comment]
Thanks @devonnuri for your contribution. I'm working on v6.0 now and I'm changing the value type to `string` (pure markdown) instead of the shady `MdeState` and I'm keeping an internal cache that I invalidate when the passed string is different from the previous one. This should make everyone's life easier. WIP PR: https://github.com/andrerpena/react-mde/pull/128

**Comment by vishrutshah** [comment]
@andrerpena This is great! Any ETA on when are you planning to release `v6.0` ?

**Comment by Sexual** [comment]
@andrerpena Still planned for the 30th? This is something we desperately need.

**Comment by lukecamelo** [comment]
@Sexual For real.

I've been using the hack @devonnuri posted but now I have a case where it doesn't work, I'm even getting strange errors. I have an edit button for comments, it brings up a modal with an instance of ReactMde in it and for the life of me I cannot get it to update.

**Comment by andrerpena** [comment]
I'm sorry @Sexual and @lukecamelo. I'm past my own deadline. I'll try to deliver a `6.0-beta` tomorrow, October 3rd. One problem is the documentation.

Unfortunately there will be a bunch of API and visual changes to it. Check more here: https://github.com/andrerpena/react-mde/issues/136

**Comment by andrerpena** [comment]
This problem should be fixed on version 6.0.0-alpha.1. I'm sorry but this contains a number of [breaking changes](https://github.com/andrerpena/react-mde/blob/master/docs-md/ChangeLogMigrating.md#from-5-to-6). Please let me know if it does not work for you.
